gstreamer-base/src/flow_combiner.rs

```diff
@@ -11,11 +16,15 @@ use std::mem;
 use std::ptr;
 
 glib_wrapper! {
-    pub struct FlowCombiner(Shared<ffi::GstFlowCombiner>);
+    pub struct FlowCombiner(Boxed<ffi::GstFlowCombiner>);
```

Comment:
It still has to be Shared, the boxed copy/free functions wrap around reference counting (unfortunately).

Reply:
I must admit I have difficulties being sure of the implications since it involves concepts from multiple frameworks that I'm still discovering. I gave a look at these types and it seemed to me that `g_boxed_copy` will actually perform a copy of the boxed object. Is it what we intend to do for `GstFlowCombiner`s? It seems to contradict the `SharedMemoryManager` trait.

Is this what this should look like:
``` rust
glib_wrapper! {
    pub struct FlowCombiner(Shared<ffi::GstFlowCombiner>);

    match fn {
        ref => |ptr| {
            gobject_ffi::g_boxed_copy(ffi::gst_flow_combiner_get_type(), ptr as *mut _) as *mut ffi::GstFlowCombiner
        },
        unref => |ptr| {
            gobject_ffi::g_boxed_free(ffi::gst_flow_combiner_get_type(), ptr as *mut _)
        },
        get_type => || ffi::gst_flow_combiner_get_type(),
    }
}
```

Reply:
Oh! I think I missed the last part of your comment (again...), the one about reference counting for Boxed types. I'll push a squashed version, hopefully correct this time.